### ISSUE TITLE
Search on Objects - network tuple

### ISSUE THREAD

**Issue by Kafeine** [MISP/MISP#3484]
| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Suggestion
| MISP version / git hash   | 2.4.90

### Actual behavior
Objects are awesome, but when it comes to search for network tuple (domain|ip) we have no clue on the object and additionnal context.

Example : for those two objects:
![image](https://user-images.githubusercontent.com/1312794/42737734-c86fe532-886f-11e8-88d0-bef875de8e7c.png)

Searching for the IPs we get this :
![image](https://user-images.githubusercontent.com/1312794/42737731-c15bb7da-886f-11e8-9dc1-326b272eb211.png)

While doing the same with  IPs stored as simple domain|ip attribute the search gives this:

![image](https://user-images.githubusercontent.com/1312794/42737692-7bbefc50-886f-11e8-9b9b-14a982f77bd3.png)

I wonder if it would be possible to insert the object description (or maybe other accessible data) doing search for attribute integrated in objects, maybe as data that can be expanded ?

**Comment by iglocska** [comment]
Yeah this has been on my todo list for way too long, will try to get something working this coming week!
